actionview/app/assets/javascripts/rails-ujs/utils/ajax.coffee

```diff
@@ -14,7 +14,9 @@ AcceptHeaders =
 Rails.ajax = (options) ->
   options = prepareOptions(options)
   xhr = createXHR options, ->
-    response = processResponse(xhr.response, xhr.getResponseHeader('Content-Type'))
+    xhrResponseObject = xhr.response
+    xhrResponseObject = xhr.responseText if !xhrResponseObject? and typeof xhr.responseText == 'string' # for IE9
+    response = processResponse(xhrResponseObject, xhr.getResponseHeader('Content-Type'))
```

Comment:
If you're happy to drop type check, sure. That came from jQuery and appeared to be a workaround of other browser nasties, although I didn't dig too deep. Done.